front/src/modules/settings/profile/components/ConfirmationModal.tsx

```diff
@@ -0,0 +1,54 @@
+import { AnimatePresence, LayoutGroup } from 'framer-motion';
+
+import { ButtonVariant } from '@/ui/button/components/Button';
+import { Modal } from '@/ui/modal/components/Modal';
+import { Section, SectionAlignment } from '@/ui/section/components/Section';
+import { H1Title, H1TitleFontColor } from '@/ui/typography/components/H1Title';
+
+import {
+  DeleteModalProps,
```

Comment:
I think we should try to make something more generic:

- keep the ConfirmationModal you've created, move the DeleteModalProps (=> ConfirmationModalProps), StyledCenteredButton, StyledDeleteButton (==> StyledConfirmationButton) styling to Confirmation Modal component
- add optional props for: confirmationValue, confirmationPlaceholder
- if confirmationValue is setup, display the Text input and activate Confirm button only if it does match
- move this to the @/ui/modal/confirmation/components
- replace DeleteModal usages by the new modal